sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -487,14 +514,22 @@ def to_open_api(self) -> OpenApiModel:
 @dataclass
 class StandardRouterEnsemblerConfig(RouterEnsemblerConfig):
     def __init__(
-        self, experiment_mappings: List[Dict[str, str]], fallback_response_route_id: str
+        self,
+        fallback_response_route_id: str,
+        experiment_mappings: Optional[List[Dict[str, str]]] = None,
+        route_name_path: Optional[str] = None,
     ):
         """
         Method to create a new standard ensembler
 
+        :param route_name_path: configured route name path that points to the route name within a given treatment config
         :param experiment_mappings: configured mappings between routes and treatments
+        :param fallback_response_route_id: configured final response route to be used as a fallback
         """
-        self.experiment_mappings = experiment_mappings
+        self.route_name_path = route_name_path
+        self.experiment_mappings = (
+            experiment_mappings  # sif experiment_mappings else None
```

Comment:
Why not just `self.experiment_mappings = experiment_mappings`? And, I think the comment is self-explanatory.

Reply:
Ahah thanks for catching the comment that's not meant to be a comment 😅